### ISSUE TITLE
[Feature request] Adds auto-start

### ISSUE THREAD

**Issue by plakhin** [Neway6655/Sublime-Pomodoro#12]
It would be nice to have ability to enable auto-start of work session when Sublime launches.

**Comment by Neway6655** [comment]
Hi plakhin,

this feature was added, by default I have disabled auto-start, you can enabled it in your settings.

Pls have a try and I will close this issue later.

**Comment by plakhin** [comment]
Hi!
Just updated plugin via package control and updated key bindings. When trying to start with ctrl+shift+alt+p - I'm getting this error in console:
```
Traceback (most recent call last):
  File "C:\Users\stan\Dropbox\Temp\Sublime Text 3\sublime_plugin.py", line 812, in run_
    return self.run(edit, **args)
TypeError: run() got an unexpected keyword argument 'autoStart'
```

**Comment by Neway6655** [comment]
Oh, that's a configuration bug, please update the plugin to version 2017.10.10.02.11.21

**Comment by plakhin** [comment]
Updated. Getting the same error if there is `"autoStart": true` parameter in keybinding.

**Comment by Neway6655** [comment]
Have you configured user's key binding settings? If yes, please remove "autoStart" and try again.

**Comment by Neway6655** [comment]
Please remove "autoStart": true parameter in key binding settings. 

This is a configuration parameter in Pomodoro.sublime-settings file.

**Comment by plakhin** [comment]
Already did. But I cannot find  Pomodoro.sublime-settings file, even after reinstalling the plugin.

**Comment by Neway6655** [comment]
You can download the Pomodoro.sublime-settings file from this code repo first.

I will check the upgrade/installation problem later.

Thanks!

**Comment by plakhin** [comment]
Where should I place it?

**Comment by Neway6655** [comment]
Put the setting file in the Sublime-Pomodoro package,

e.g. in Mac OS: Open the folder from Sublime "Sublime Text->Preferences->Browse Packages", then you will find a "Sublime-Pomodoro", please put the setting file in this folder.

**Comment by plakhin** [comment]
I'm using Sublime Text 3 build 3143 on Windows 10. The only folder which contains anything about Sublime-Pomodoro is `Sublime Text 3\Data\Installed Packages` with `Pomodoro.sublime-package` file. Putting `Pomodoro.sublime-settings` file in this folder gives no effect.

**Comment by Neway6655** [comment]
I have added a Menu setting file, please update with latest version or reinstall it, then you can change the settings in "Preferences->Package Settings->Package Control->Pomodoro".

**Comment by plakhin** [comment]
I've reinstalled the plugin. Now there are Settings in menu as you described, but both files (default settings and user settings) are empty. I've placed array with config to user settings and autostart is working fine now. Thanks. But I think it makes sense to place default config in default settings by default and allow to bind keys without providing array with settings (at the moment plugin will not start if "args":{} with 4 params is not provided in key bindings), but if settings are provided in key bindings they may override user settings which override default settings.

**Comment by donbalon4** [comment]
hey @Neway6655 , I agree with @plakhin . I can work on it, if you want it :D

**Comment by Neway6655** [comment]
@donbalon4 sure, welcome to contribute it together, thanks.

**Comment by donbalon4** [comment]
@Neway6655 , I think this issue should be closed due to #14

**Comment by Neway6655** [comment]
Thanks, @donbalon4
